crates/api/src/consts.rs

```diff
@@ -0,0 +1,26 @@
+/// Maximum length for generic name fields persisted in the database
+pub const MAX_NAME_LENGTH: usize = 255;
+
+/// Maximum length for generic description fields
+pub const MAX_DESCRIPTION_LENGTH: usize = 2048;
+
+/// Maximum length for email fields
+pub const MAX_EMAIL_LENGTH: usize = 255;
+
+/// Maximum length for organization/system prompts
+pub const MAX_SYSTEM_PROMPT_LENGTH: usize = 8192;
```

Comment:
```suggestion
pub const MAX_SYSTEM_PROMPT_LENGTH: usize = 64 * 1024;
```

8K system prompt is not enough in some scenarios.